platforms/shared/configuration/setup-k8s-environment.yaml

```diff
@@ -100,6 +100,17 @@
     when: 
     - network.type == 'fabric'
     - network.env.type == 'operator'
+
+  # Deploy prometheus server
+  - name: Deploy prometheus server
+    include_role:
+      name: setup/prometheus
+    vars:
+      prometheus_port: "{{ network.prometheus.port }}"
+    when: 
+    - network.prometheus.enabled == "true"
```

Comment:
Use "network.prometheus is defined and network.prometheus.enabled "

Reply:
Fixed.